docs/docs/gatsby-project-structure.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Gatsby Project Structure
+---
+
+This is a high-level introduction of the top-level files and directories you'll see in a Gatsby project.
+
+1. `/node_modules`: The directory where all of the modules of code that your project depends on (npm packages) are automatically installed.  
+2. `/src`: This directory will contain all of the code related to what you will see on the front-end of your site (what you see in the browser), like your site header, or a page template. “Src” is a convention for “source code”.
```

Comment:
The explanation of what "src" means, while useful, doesn't quite fit the style of the rest of this documentation, which uses lots of jargon that is not defined. I assume "src" is at least as widely known as "git"? If so, let's just leave that explanation out. Everything else that is explained (like prettier) seems justified to me, since some things might not be that universal.

Reply:
What do you think of having the fuller more explanatory language on the tutorial page, and shortening it on the docs page?

Reply:
That sounds fine for now. This just reminds me of how helpful it will be to clearly define our audience @KyleAMathews because we've gone back and forth on whether we are aiming to accommodate beginners